dashboard/db/migrate/20240806035905_rename_cap_columns_in_users.rb

```diff
@@ -0,0 +1,6 @@
+class RenameCAPColumnsInUsers < ActiveRecord::Migration[6.1]
+  def change
+    rename_column :users, :cap_state, :cap_status
+    rename_column :users, :cap_state_date, :cap_status_date
+  end
+end
```

Comment:
@code-dot-org/infrastructure Do we expect this to have a significant performance impact, or be too slow for a regular migration during the DTP?

Reply:
This should be just fine! We get column and index rename operations for free on our current version of MySQL, and fortunately I already had a clone of the production db lying around I could use to verify:

```
ubuntu@adhoc-cap-data-migration-testing:~/adhoc/dashboard$ RAILS_ENV=adhoc RACK_ENV=adhoc bundle exec rake db:migrate
GetSecretValue: adhoc/cdo/recaptcha_site_key
GetSecretValue: adhoc/cdo/recaptcha_secret_key
WARNING: MYSQL_OPT_RECONNECT is deprecated and will be removed in a future version.
WARNING: MYSQL_OPT_RECONNECT is deprecated and will be removed in a future version.
WARNING: MYSQL_OPT_RECONNECT is deprecated and will be removed in a future version.
== 20240806035905 RenameCAPColumnsInUsers: migrating ==========================
-- rename_column(:users, :cap_state, :cap_status)
   -> 0.8567s
-- rename_column(:users, :cap_state_date, :cap_status_date)
   -> 0.7402s
== 20240806035905 RenameCAPColumnsInUsers: migrated (1.5972s) =================

WARNING: MYSQL_OPT_RECONNECT is deprecated and will be removed in a future version.
```